Kwenta/kwenta
# Integrated the socket plugin

## Description

1. Integrate the Socket widget into our onboarding flow (when user has less than 50 sUSD to deposit)
2. Style the Socket widget following our current theme design

## Related issue
#1657 

## Motivation and Context

Make the onboarding process more smoothly

## How Has This Been Tested?
### Test Case 1 - No wallet connected
Click on the `Deposit Margin` and the normal `Deposit Margin` modal pop-up.

### Test Case 2 - User with more than 50 sUSD
Click on the `Deposit Margin` and the normal `Deposit Margin` modal pop-up.

### Test Case 3 - User with less than 50 sUSD
1. Click on the `Deposit Margin` and the `Deposit` modal pop-up with the `Bridge & Swap` button 
2. Click on the `Bridge & Swap`, and the user can see the Socket widget

## Screenshots (if appropriate):
### Light theme
![image](https://user-images.githubusercontent.com/4819006/210596369-ca48d194-7e43-4515-9ca8-ca60943ee808.png)
![image](https://user-images.githubusercontent.com/4819006/210575270-af6fefec-918c-4eef-add3-7aeb68dc8850.png)
### Dark theme
![image](https://user-images.githubusercontent.com/4819006/210596232-722ed75c-bbb5-4cbb-987b-d9a3b4e241c3.png)
![image](https://user-images.githubusercontent.com/4819006/210595377-0964836b-be1f-49b7-a099-ffcec106eaed.png)
